bagisto/bagisto
# Issue Fixed #6886 (Locale Dropdown Overlapping on Mobile View issue fixed)

## Issue Reference

#6886 
## Description

Issue Fixed => Locale Dropdown Overlapping on Mobile View 
## How To Test This?

Steps To Test--
1. Log in as an admin on mobile.
2. Click on products inside the catalog tab.
3. Try to change the app locale.

Check Here (Issue Fixed)
![image](https://user-images.githubusercontent.com/42333336/195353733-fdaae4d8-a485-4ca2-9608-d22018ad5c7b.png)

## Documentation
